src/app/blog/components/post-list/post-list.component.html

```diff
@@ -45,7 +45,7 @@ <h1 data-aos="fade-in-up" data-aos-offset="100" class="text-primary">Nos Actuali
                 <div class="blig-item">
                   <article>published Date: {{blog.publishedDate}}</article>
                   <!-- here is to display the tooltip message on top of the page -->
-                  <h4 pTooltip="this is number of comments" tooltipPosition="top"><i class="pi pi-comments"></i> {{blog.comments.length}}</h4>
+                  <h4 ><i class="pi pi-comments" pTooltip="this is the total number of comments" tooltipPosition="top"></i> {{blog.comments.length}}</h4>
```

Comment:
Why do you use h4 element hier ?
What i know: **h4 elemen**t stretch his contain over teh whole width --> that's why tootip is far away
by using **span element** you have not stress with the content  and tooltip

Reply:
h4 element here is to display the pi-comments in bold form and also in small character.
no the topic now if just on the top of the comment icon